internal/capability/auditor.go

```diff
@@ -10,30 +10,29 @@ import (
 // It has methods to create a workqueue with all the package and audit requirements for a
 // particular audit run
 type Auditor interface {
-	BuildWorkQueueByCatalog(opts operator.OperatorCheckOptions) error
```

Comment:
+1 good that you made this internal didn't think of it 😅

Reply:
Yeah. I believe it's better like that.